operators/o2ims-operator/README.md

```diff
@@ -0,0 +1,266 @@
+# Nephio O-RAN O2 IMS Operator
+
+This operator implements O-RAN O2 IMS for K8s based cloud management. 
+
+## How to start
+
+### Development Requirements:
+
+- Python3.10
+- requirements.txt installed in development environment
+
+### Nephio Management Cluster Requirements:
+
+- 6 vCPU
+- 10Gi RAM
+
+## Create Development Environment
+
+### Including Nephio mgmt Cluster
+
+The following will create a kind cluster and install required components such as:
+- Porch
+- ConfigSync
+- Gitea (available at `172.18.0.200:3000`)
+- MetalLB and MetalLB Sandbox Environment
+- CAPI
+- ConfigSync and RootSync objects to create clusters
+
+It will also configure a secret which the operator can use for development purposes (when running the operator in non-containerize environments). It creates a pod and appends the `porch-controllers` service account token and redirects it from `/var/run/secrets/kubernetes.io/serviceaccount/token` to `/tmp/porch-token`.
+
+
+```bash
+# Get the repository
+git clone https://github.com/nephio-project/nephio.git
+cd o2ims-operator
+# Create a virtual environment
+virtualenv venv -p python3
+source venv
+# Install requirements
+pip install -r requirements.txt
+# Set kernel parameters (run these commands after system restart or when new VM/system is created)
+sudo sysctl -w fs.inotify.max_user_watches=524288
+sudo sysctl -w fs.inotify.max_user_instances=512
+sudo sysctl -w kernel.keys.maxkeys=500000
+sudo sysctl -w kernel.keys.maxbytes=1000000
+# Get the CRD from the Nephio API repo and place it in o2ims-operator/config/crd/bases/
+curl --create-dirs -O --output-dir ./config/crd/bases/ https://raw.githubusercontent.com/nephio-project/api/refs/heads/main/config/crd/bases/o2ims.provisioning.oran.org_provisioningrequests.yaml
```

Comment:
@arora-sagar will modify L47 as below
`kubectl create -f https://raw.githubusercontent.com/nephio-project/api/refs/heads/main/config/crd/bases/o2ims.provisioning.oran.org_provisioningrequests.yaml
 `

Reply:
I updated the text instead.